public/src/client/topic/fork.js

```diff
@@ -17,24 +17,26 @@ define('forum/topic/fork', ['components', 'postSelect'], function (components, p
 	}
 
 	function onForkThreadClicked() {
-		app.parseAndTranslate('partials/fork_thread_modal', {}, function (html) {
-			forkModal = html;
+		if(!$('.fork-topic-card').length) {
```

Comment:
Isn't it easier to just do
```
if (forkModal) {
    return;
}
```

Reply:
I'm just checking to see if theres already one in the DOM, surely forkModal is always true, as its just getting the HTML to display?

Disclaimer: quite possible i'm talking rubbish ;)

Reply:
`forkModal` is undefined until the modal is opened, and it is set to null when it is closed, so it can be used to allow only 1 modal.

Reply:
makes sense. I'll update this PR